### DISCUSSION TITLE
same command lines but different output (Noob)

### DISCUSSION THREAD

**Discussion by aput** [ImageMagick/ImageMagick#7445] [Help] [answered]
### ImageMagick version

7.1.1-34-Q16-x64-dll

### Operating system, version and so on

WIN10 64-bit

### Description

Only just getting started with ImageMagic and this puzzels me:
Why is it that 
```
magick wizard: -rotate -69 -crop 200x250+100+100 outfile1.gif
```
and 
```
magick wizard: -rotate -69 tmpfile.gif
magick tmpfile.gif -crop 200x250+100+100 outfile2.gif
```

both give different results when all I did is make an intermediate file?

**Comment by GeeMack** [comment, marked as answer]
With your first command the "-rotate" operation adjusts the X and Y positions of the paging geometry, moving point 0,0 to somewhere other than the top left corner. Then the crop starts calculating from that location.

With the second pair of commands, IM resets the paging geometry when saving to the temporary GIF file. Then in the second part it gets cropped from the actual top left, position 0,0 in the temp file.

Adding "+repage" after the rotation in your first command will make it behave just like the second pair of commands...

    magick wizard: -rotate -69 +repage -crop 200x250+100+100 outfile1a.gif

**Comment by aput** [comment]
@GeeMack, thank you for the quick answer!
